demos/demos.css

```diff
@@ -0,0 +1,111 @@
+
+/* http://meyerweb.com/eric/tools/css/reset/
```

Comment:
This is actually an incorrect copyright header. Instead we should say something like:

`Adapted from ... <link to ...>`.

Also, I'd prefer it if we used [normalize.css](http://necolas.github.io/normalize.css/) instead for [reasons listed here](http://stackoverflow.com/questions/6887336/what-is-the-difference-between-normalize-css-and-reset-css) and imported it using [cdnjs](https://cdnjs.com/libraries/normalize) via `@import`.